README.md

```diff
@@ -1,7 +1,5 @@
 **[ChooseALicense.com](https://choosealicense.com)** aims to provide **accurate**, **non-judgmental**, and **understandable** information about popular **open source licenses** in order to **help people make informed decisions** about the projects they start, maintain, contribute to, and use.
 
-[![Build Status](https://travis-ci.org/github/choosealicense.com.svg?branch=gh-pages)](https://travis-ci.org/github/choosealicense.com)
```

Comment:
Didn't replace this with the GitHub version, but this badge is currently "broken" since Travis changed to supporting bitbucket, and the "github" in the URL now represents the site https://travis-ci.org/github/choosealicense.com